lib/app.ex

```diff
@@ -0,0 +1,36 @@
+defmodule App do
```

Comment:
it's generally a good idea to make your App name match your project name, so I would consider calling it `AntColony.Application` and put it in the `ant_colony` directory